app/views/govuk_publishing_components/components/_radio.html.erb

```diff
@@ -28,7 +36,8 @@
   radio_classes << "govuk-radios--inline" if inline
 
   legend_classes = %w(govuk-fieldset__legend)
-  legend_classes << "govuk-fieldset__legend--#{heading_size}"
+  legend_classes << "govuk-fieldset__legend--#{size}"
+  legend_classes << "gem-c-title" if is_page_heading 
```

Comment:
Sorry, just noticed, this class `gem-c-title` is from another component. I don't think components should be using each others classes like this - could this be a `govuk-heading` class instead?

Reply:
This line is no longer needed